File: app/src/main/java/com/ratik/uttam/model/PhotoType.java
package com.ratik.uttam.model;

/**
 * Created by Ratik on 02/02/18.
 */

public enum PhotoType {
    FULL(0), REGULAR(1), THUMB(2);

    private final int value;

    PhotoType(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }
}

File: app/src/main/kotlin/com/ratik/uttam/domain/model/PhotoType.kt
package com.ratik.uttam.domain.model

/** Created by Ratik on 02/02/18. */
@Deprecated("Redundant, just directly use [Photo] instead")
enum class PhotoType(val value: Int) {
  FULL(0),
  REGULAR(1),
  THUMB(2),
}
